WCSim.cc

```diff
@@ -114,6 +126,7 @@ int main(int argc,char** argv)
     G4String command = "/control/execute ";
     G4String fileName = argv[1];
 
+    file_exists(fileName.c_str());
```

Comment:
What happens if you just run WCSim with a non-existent file name on the command line now?  I thought it already didn't work...